<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsHistory</class>
 <widget class="QWidget" name="ResultsHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #e6f0fa; /* Fond bleu tr√®s clair */
    font-family: 'Segoe UI', sans-serif;
    font-size: 14px;
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* Main title */
QLabel#titleLabel {
    font-size: 24px;
    font-weight: bold;
    color: #1F3C88;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
                              stop:0 rgba(255,255,255,0.9), 
                              stop:1 rgba(245,249,255,0.8));
    border-bottom: 2px solid #2A5D9F;
    padding: 12px 20px;
    margin-bottom: 15px;
}</string>
   </property>
   <property name="text">
    <string>             üîç Results History</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>651</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    font-size: 18px;
    font-weight: bold;
    color: #607D8B;
    border: 1px solid #B0BEC5;
    border-radius: 4px;
    margin-top: 20px;
    padding-top: 25px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 5px;
    left: 10px;
}</string>
   </property>
   <property name="title">
    <string>üîç Results History</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>551</width>
      <height>304</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="dateFilterLabel">
       <property name="styleSheet">
        <string notr="true">QLabel#dateFilterLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
       </property>
       <property name="text">
        <string>Date :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxResponsible">
       <property name="styleSheet">
        <string notr="true">QComboBox {
    border: 1px solid #1F3C88;
    border-radius: 4px;
    padding: 6px;
    background: #F5F9FF;
}</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QPushButton" name="backButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#backButton {
	background-color: #2a5d9f;
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white 
}

QPushButton#backButton:hover {
    background-color: #1f497d;
}

</string>
       </property>
       <property name="text">
        <string>‚¨ÖÔ∏è Back</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEditFilter">
       <property name="styleSheet">
        <string notr="true">QDateEdit {
    border: 1px solid #1F3C88;
    border-radius: 4px;
    padding: 6px;
    background: #F5F9FF;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidgetResults">
       <property name="styleSheet">
        <string notr="true">/* Style pour la table enti√®re */
QTableWidget {
    border: 1px solid #113f7c;
    border-radius: 3px;
    gridline-color: #c0c0c0;
    font-family: &quot;Segoe UI&quot;;
    font-size: 13px;
}

/* Style pour l'en-t√™te des colonnes */
QHeaderView::section {
    background-color: #113f7c;
    color: white;
    padding: 8px 12px;
    border: none;
    font-weight: bold;
    font-size: 12px;
}

/* Style pour les cellules de la table */
QTableWidget QTableCornerButton::section {
    background: #113f7c;
    border: none;
}

QTableView::item {
    padding: 6px;
    border-bottom: 1px solid #e0e0e0;
}

/* Style pour les cellules s√©lectionn√©es */
QTableView::item:selected {
    background-color: #113f7c;
    color: white;
}

/* Style au survol */
QTableView::item:hover {
    background-color: #d0e0f0;
}

/* Style altern√© pour les lignes */
QTableView::item:nth-child(even) {
    background-color: #f8f8f8;
}

QTableView::item:nth-child(odd) {
    background-color: white;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="responsibleLabel">
       <property name="styleSheet">
        <string notr="true">QLabel#responsibleLabel {
    font-size: 18px;
    font-weight: bold;
    color: #113f7c; /* Bleu profond */
}
</string>
       </property>
       <property name="text">
        <string>Responsible :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="viewDetailsButton">
         <property name="styleSheet">
          <string notr="true">QPushButton#viewDetailsButton {
	background-color: #113f7c; 
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white;
}

QPushButton#viewDetailsButton:hover {
    background-color: #0e3262;
}
</string>
         </property>
         <property name="text">
          <string>üìã View Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="exportPDFButton">
       <property name="styleSheet">
        <string notr="true">QPushButton#exportPDFButton {
	background-color: #113f7c; 
    padding: 8px 16px;
    border-radius: 6px;
	font-size: 18px;
    font-weight: bold;
    color: white;
}

QPushButton#exportPDFButton:hover {
    background-color: #0e3262;
}
</string>
       </property>
       <property name="text">
        <string>üì§Export PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
